apis/pkg/v1/package_runtime_types.go

```diff
@@ -46,6 +46,19 @@ type PackageRevisionRuntimeSpec struct {
 	// certificates of the Provider.
 	// +optional
 	TLSClientSecretName *string `json:"tlsClientSecretName,omitempty"`
+
+	// ServiceAccountRef references the ServiceAccount resource that is used
+	// by the provider revision's runtime.
```

Comment:
it this saying "provider" because we only expect to use and respect it for Providers?

Reply:
I'd say that's not the case, given that you updated also functions to use that. However, didn't we decide Functions should not have to have access to the kubernetes API, so should we just make it a no-op for functions? 🤔

Reply:
Thanks for the suggestion, I believe those changes are gone in the latest state.